### PR TITLE
users can add meals to their weekly planner and access it in their dashboard

### PR DESCRIPTION
**Pull request by OluwadaraO**
Users are able to add a meal to their weekly planner and delete it as well.
This is a demo of a user using this feature: https://www.loom.com/share/7744c8dca03b4893a328cba7a634eee4?sid=c15b3269-9708-483d-a845-40ef8b3dc0e3

### CODE REVIEW

File: culinary-canvas/capstone-project/frontend/my_capstone_project/src/HomePage.jsx [hunk 1]
```diff
@@ -21,6 +22,7 @@ function HomePage() {
     "cookies",
     "cheese",
     "salad",
+    "pasta",
```
Comment (ayoubmansaratwork) [inline culinary-canvas/capstone-project/frontend/my_capstone_project/src/HomePage.jsx:25]: Question - can you help me explain what this `const queries` does? Why do we have a hardcoded list of foods? What if a user wants to have `pizza` which is not in that list?

File: culinary-canvas/capstone-project/frontend/my_capstone_project/src/MealPlannerDashboard.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                        <p>{plan.recipeName}</p>
+                        <button onClick={() => handleDelete(plan.id)}>Delete</button>
+                        <a href={plan.recipeUrl} target="_blank" rel="noopener noreferrer">
+                            <button>View Recipe</button>
+                        </a>
+                    </div>
+                ))
+            ): (
+                <p>No meal plans yet. Add some recipes to your planner by clicking on the plus sign of that recipe!</p>
+            )}
```
Comment (ayoubmansaratwork) [inline culinary-canvas/capstone-project/frontend/my_capstone_project/src/MealPlannerDashboard.jsx:56]: Nice job adding fallback handling for this.

**Review by ayoubmansaratwork — APPROVED** [review]
Changes look good.

Can you also test what happens if a user adds a recipe to "Monday, Breakfast" for example, and then tries to add another recipe to that same time? Does it safely say to the user that there is an error (or maybe better to tell them that they already have a recipe planned for that day+time)?

File: culinary-canvas/capstone-project/frontend/my_capstone_project/src/HomePage.jsx (hunk 1 above)
Comment (OluwadaraO) [reply culinary-canvas/capstone-project/frontend/my_capstone_project/src/HomePage.jsx:25]: The API doesn't allow me to use the keyword random so i just hardcoded a list of foods when I started fetching the recipes from the API

**Pull request merged**